src/roles/subuid_subgid_check/tasks/main.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Check /etc/subuid for current user
+  ansible.builtin.command: grep "^{{ ansible_user_id }}:" /etc/subuid
```

Comment:
Should this use [lineinfile](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/lineinfile_module.html) in check mode instead?